core/src/main/scala/org/apache/spark/util/ThreadUtils.scala

```diff
@@ -156,4 +156,11 @@ private[spark] object ThreadUtils {
         result
     }
   }
+
+  /**
+   * Construct a new ForkJoinPool with a specified max parallelism.
+   */
+  def newForkJoinPool(maxThreadNumber: Int): scala.concurrent.forkjoin.ForkJoinPool = {
```

Comment:
Yes, if it's more than calling a constructor it may be worth it but not to wrap up "new ForkJoinPool" as "newForkJoinPool = new ForkJoinPool"